migrations/versions/201509301444_f9bb2a03ceb_add_registration_tables.py

```diff
@@ -42,6 +42,7 @@ def upgrade():
         sa.Column('moderation_enabled', sa.Boolean(), nullable=False),
         sa.Column('notifications_enabled', sa.Boolean(), nullable=False),
         sa.Column('recipients_emails', postgresql.ARRAY(sa.String()), nullable=False),
+        sa.Column('eticket_enabled', sa.Boolean(), nullable=False),
```

Comment:
I just appended it to the list of the columns so it matches the order in which the columns have been added. Anyway,  i can place it somewhere else.